### PR TITLE
feat: add query parameters to dump CLI log file into stdout

### PR DESCRIPTION
**Pull request by gabriel-farache**
feat: add query parameters to dump CLI log file into stdout

Related to https://github.com/konveyor/move2kube-api/issues/159

### CODE REVIEW

File: internal/filesystem/filesystem.go [hunk 1]
```diff
@@ -2313,6 +2315,9 @@ func (fs *FileSystem) runPlan(currentRunDir string, currentRunConfigPaths []stri
 	default:
 		logrus.Debug("ctx not closed")
 	}
+	if dumpCliLogs {
+		dumpLogFileToStdOut(currentRunDir, logFile)
```
Comment (HarikrishnanBalagopal) [inline internal/filesystem/filesystem.go:2319]: return error from the function and check the error here, logging if necessary

**Review by HarikrishnanBalagopal — COMMENTED** [review]
Some minor comments

File: internal/filesystem/filesystem.go (hunk 1 above)
Comment (gabriel-farache) [reply internal/filesystem/filesystem.go:2319]: I returned the error produced by `os.ReadFile(logFilePath)`

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM

**Pull request merged**